src/main/java/edu/cornell/kfs/ksr/document/validation/impl/SecurityGroupRule.java

```diff
@@ -0,0 +1,167 @@
+package edu.cornell.kfs.ksr.document.validation.impl;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.commons.collections.CollectionUtils;
```

Comment:
Now that this class is using CollectionUtils, it should be updated to use the collections4 variant of it.

Reply:
updated